Dockerfile

```diff
@@ -229,7 +229,8 @@ RUN export COMPOSER_HOME=/composer && \
     composer install --working-dir=/opt/kimai --optimize-autoloader && \
     composer clearcache && \
     composer require --working-dir=/opt/kimai laminas/laminas-ldap && \
-    chown -R www-data:www-data /opt/kimai
+    chown -R www-data:www-data /opt/kimai && \
+    sed "s/128M/256M/g" /usr/local/etc/php/php.ini-development > /usr/local/etc/php/php.ini
```

Comment:
I would feel safer if you replace the complete line `memory_limit=128M` with `memory_limit=256M`.

Is command line and fpm php using the same php.ini?

Reply:
There is only one php.ini in the fpm image